### ISSUE TITLE
unable to add require ssl to mysql user

### ISSUE THREAD

**Issue by andredublin** [ansible/ansible-modules-core#844]
ansible version 1.8.2
Here is my error

``` shell
msg: invalid privileges string: Invalid privileges specified: frozenset(['REQUIRESSL'])
```

here is the command that causes the error

``` yaml
mysql_user: name=andre append_privs=yes priv=*.*:REQUIRESSL state=present
```

Also the documentation has an example that has `append=true` instead of `append_privs=yes`

**Comment by abadger** [comment]
I believe this has been fixed already by adding REQUIRESSL here: https://github.com/ansible/ansible-modules-core/blob/devel/database/mysql/mysql_user.py#L165

The fix should be in 18.3/1.8.4 so upgrading should fix your problem.

**Comment by abadger** [comment]
Documentation fixed to use append_privs instead of appen: https://github.com/ansible/ansible-modules-core/commit/addca40604a98e226184529a756b5d318102367d

# Closing This Ticket

Hi!

We believe recent commits (likely detailed above) should resolve this question or problem for you.

This will also be included in the next major release.

If you continue seeing any problems related to this issue, or if you have any further questions, please let us know by stopping by one of the two mailing lists, as appropriate:
- https://groups.google.com/forum/#!forum/ansible-project - for user questions, tips, and tricks
- https://groups.google.com/forum/#!forum/ansible-devel - for strategy, future planning, and questions about writing code

Because this project is very active, we're unlikely to see comments made on closed tickets, but the mailing list is a great way to ask questions, or post if you don't think this particular
issue is resolved.

Thank you!
